dot/parachain/backing/per_relay_parent_state.go

```diff
@@ -416,3 +423,35 @@ func executorParamsAtRelayParent(
 	// https://github.com/paritytech/polkadot-sdk/blob/7ca0d65f19497ac1c3c7ad6315f1a0acb2ca32f8/polkadot/node/subsystem-util/src/lib.rs#L241-L242
 	return parachaintypes.ExecutorParams{}, nil
 }
+
+func getPovFromValidator(
+	subSystemToOverseer chan<- any,
+	chRelayParentAndCommand chan relayParentAndCommand,
+	relayParent common.Hash,
+	candidateHash parachaintypes.CandidateHash,
+	attesting *attestingData,
+) (*parachaintypes.PoV, error) {
```

Comment:
We don't need this to be a pointer. getPovFromValidator returns a pointer, but 
- we are not changing the value of this PoV anywhere
- we are immediately getting a value from this pointer and using that value everywhere

Just return a value instead of pointer.